src/plugins/legacy/polygonRoi/polygonRoiToolBox.cpp

```diff
@@ -611,6 +664,95 @@ void polygonRoiToolBox::saveBinaryImage()
     viewEventFilter->saveMask();
 }
 
+void polygonRoiToolBox::addLabelName()
+{
+    if ( structureList->count()==colorsList.size())
+    {
+        displayMessageError("Unable to create more label. Maximum size achieved");
+        plusButton->setDisabled(true);
+        return;
```

Comment:
in order to avoid `return` in functions when it could be avoided, you can change this `return` to an `else` after this `if`. The code would be clearer :)